ipynb/tests/Generic_EAS_Tests.ipynb

```diff
@@ -0,0 +1,533 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "collapsed": true
+   },
+   "source": [
+    "# Run Generic Automated EAS tests\n",
```

Comment:
It would be good to have more information about what the tests report and how the evaluation is done. As you say below "These tests make estimation of the energy efficiency of task placements.. " + "by.. ". It would also be nice to make clear the distinction between the estimated observed power consumption (estimated based on the energy model provided and the observed task placement decisions made) and the expected power consumption (estimated based on the energy model provided and the expected optimal task placements). This can be done either here at the beginning or further in the notebook where the estimated and expected data frames are obtained.